lib/outdated.js

```diff
@@ -111,7 +111,12 @@ function outdated (args, silent, cb) {
         }
         output(table(outTable, tableOpts))
       }
-      cb(null, list.map(function (item) { return [item[0].parent.path].concat(item.slice(1, 7)) }))
+      var msg = [
+        'The outdated dependencies were found. Please make sure',
+        'the versions in node_module, or look over package.json',
+        'to use the most suitable semver range.'
+      ].join('\n')
+      cb(msg, list.map(function (item) { return [item[0].parent.path].concat(item.slice(1, 7)) }))
```

Comment:
Quick question – should the first arg here be an Error instead of a String? (not sure about npm's style preference – but most Node.js projects typically use Errors so that stack information is retained, among other things)